docs/reproducibility.md

```diff
@@ -8,48 +8,152 @@ The assumption here is that the original build was done using a Docker.
 
-- The quote of a running application code, get it from `https://<LEDGER-URL>/node/quotes/self` (replace <LEDGER-URL> with the URL of your ledger), e.g.:
+    ```sh
+    $ curl -k "https://<LEDGER-URL>/app/parameters" | jq -r .serviceCertificate | base64 -d > cacert.der
+    $ openssl x509 -inform der -in cacert.der > cacert.pem
+    ```
+
+- The quote of a running application code, get it from `https://<LEDGER-URL>/node/quotes` which will contain the measurements of each node in the network. They will be the same almost all of the time except when upgrading to the new version e.g.:
 
     ```sh
-    $ curl -s --cacert cacert.pem https://<LEDGER-URL>/node/quotes/self > node-quote.json
+    $ curl -s --cacert cacert.pem https://<LEDGER-URL>/node/quotes > node-quotes.json
     ```
+
 - Source code version which was used to build the application, usually found in `https://<LEDGER-URL>/app/version`. If `app/version` is ambiguous then check the build logs of the Docker image to understand which commit produced the candidate value. e.g., `fb2c496416fbab20837fedda0ba6db58d819fa5f5c1b3916062eb2fb9d889966` was built from `fd77c0c69ee890bdc2fcf6ef0c9dddb7b211e164`.
 
     ```sh
     $ curl -s --cacert cacert.pem https://<LEDGER-URL>/app/version | jq ".version"
-    "1.2.3"
+    "0.12.3-0-gaaaaaaa"
     ```
 
-### Extract measurements
+- Security policy used to verify the container image ([ccf docs](https://microsoft.github.io/CCF/main/governance/gov_api_schemas/2024-07-01.html#get--gov-service-join-policy)), it will contain image layers, e.g.:
+
+    ```sh
+    $ curl -s --cacert cacert.pem https://<LEDGER-URL>/gov/service/join-policy?api-version=2024-07-01 > service-join-policy.json
+    ```
+
+### Extract measurements from the report
 
 The quote contains the attestation report that has the necessary measurements. `Measurement` will have the launch measurement of the guest virtual machine (aka utility VM or UVM) which was used to run the container. `Host data` will have the security policy measurement which was used to make sure no other container was launched except the one you are inspecting. More detail on implementing relying party logic can be [found in Confidential ACI scheme documentation](https://github.com/microsoft/confidential-aci-examples/blob/main/docs/Confidential_ACI_SCHEME.md#reference-info-base64).
 
 - Decode the report and save it to a file:
 
     ```sh
-    cat node-quote.json | jq -r '.raw' | base64 -d > snp-report.bin
+    $ cat node-quotes.json | jq -r '.quotes[0].raw' | base64 -d > snp-report.bin
     ```
 
-- Inspect the report to get the measurements. You could use https://github.com/virtee/snpguest to display the report details:
+- Inspect the report to get the measurements. You could use https://github.com/virtee/snpguest to display the report details, e.g.:
```

Comment:
There's also a hex2report tool in https://github.com/microsoft/confidential-sidecar-containers/blob/main/tools/get-snp-report/hex2report.c, mentioned in passing in https://github.com/microsoft/confidential-sidecar-containers?tab=readme-ov-file#fetching-an-attestion-report.

Reply:
There are no instructions how to use hex2report and the last release was more than a year ago on Github releases. I would rather not confuse the reader.